app/controllers/sessions_controller.rb

```diff
@@ -8,7 +8,10 @@ def callback
 
     if FeatureService.enabled?("allow_user_creation") && current_user.nil?
       @current_user = User.new(
-        provider: Provider.create_or_find_by(name: "DfE"),
+        provider: Provider.create_or_find_by(
+          name: "DfE",
+          dttp_id: "00000000-0000-0000-0000-000000000000",
```

Comment:
After adding the validation to the `Provider` model, this is now required. I'm not sure whether a final decision was made over how to handle this, so I've added this as a placeholder / point of discussion.

Reply:
I think that whole feature flag can be dropped